share/spice/symbolab/symbolab.js

```diff
@@ -0,0 +1,48 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_symbolab = function (api_result) {
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || !api_result.solution) {
+            return Spice.failed('symbolab');
+        }
+
+        DDG.require('mathquill', function (){
+            Spice.add({
+                id: "symbolab",
+                name: "Calculator",
+                data: api_result.solution,
+                meta: {
+                    sourceName: "Symbolab.com",
+                    sourceUrl: api_result.solution.url
+                },
+                normalize: function (data) {
+                    return {
+                        latex: data.originalQuery.replace(/ /g, "\\text{ }"),
+                        solution: data.solutionLatex.replace(/^=/, "").replace(/,/, ",\\text{ }"),
```

Comment:
It looks like when the input is LaTeX something here is breaking causing a blank output. If you strip the `\:` from the input, it works OK.

For example I copied "calculate long\:division\:\frac{x^4+6x^2+2}{x^2+5}" from Symbolab.com and it produces this output:

<img width="1439" alt="calculate_long__division___frac x 4_6x 2_2 x 2_5 _at_duckduckgo" src="https://cloud.githubusercontent.com/assets/873785/9471162/1b9609b4-4b1d-11e5-9f5d-597828d03129.png">

With the `\:` removed (and with the `sup { top: 0; }` css) it looks much better:

<img width="1438" alt="calculate_long_division__frac x 4_6x 2_2 x 2_5 _at_duckduckgo" src="https://cloud.githubusercontent.com/assets/873785/9471204/41355df0-4b1d-11e5-8d1a-5254a213f389.png">